runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3406,6 +3416,7 @@ void TR::CompilationInfo::stopCompilationThreads()
          J9Method *ramMethod = NULL;
          uint32_t romMethodOffset = 0;
          JITaaS::J9ClientStream client(getPersistentInfo());
+         incrementJITaaSNumConnectionsOpened();
```

Comment:
Thinking more about this statistics, we may lose information if some exception is thrown, exception that is not caught at this level, but is caught at an outer level.
The best solution is to include the stats in the J9ClientStream as statics. The constructor needs to increment the number of connections and the destructor needs to decrement it (or have two counters like you have now - that will show the watermark as well)